src/Raven.Server/Dashboard/ThreadsInfoNotificationSender.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Raven.Server.Background;
+using Raven.Server.NotificationCenter;
+using Raven.Server.Utils;
+using Sparrow.Collections;
+
+namespace Raven.Server.Dashboard
+{
+    public class ThreadsInfoNotificationSender : BackgroundWorkBase
```

Comment:
This is not used anywhere?

Reply:
This will be used when the studio part will be implemented